packages/ses/src/error/console.js

```diff
@@ -108,7 +108,9 @@ export const consoleWhitelist = freeze([
 // /////////////////////////////////////////////////////////////////////////////
 
 /** @type {MakeLoggingConsoleKit} */
-const makeLoggingConsoleKit = () => {
+const makeLoggingConsoleKit = loggedErrorHandler => {
+  loggedErrorHandler.resetErrorTagNum();
```

Comment:
I'm not sure I understand the utility of resetting the number here. Is the idea that we'll have multiple `console`s (i.e. "console kits") which share an underlying error handler? If so, why should the creation of a new console kit change the behavior of an existing one?

Reply:
Good catch. My layering here was bad. Fixed.

The purpose of this was only for running tests. I made this a hidden behavior of `makeLoggingConsoleKit` because I know that it is actually only used for testing. I forgot that I put it in a `src/` not `test/` directory because it can be used to make a console. Instead, I added an option here as well, `shouldResetForDebugging` that defaults to `false`. Its one caller, `throwsAndLogs` which is in a `test/` directory, calls it with that option set to `true`.